runtime/fastly/builtins/secret-store.cpp

```diff
@@ -46,7 +81,8 @@ const JSPropertySpec SecretStoreEntry::static_properties[] = {
 };
 
 const JSFunctionSpec SecretStoreEntry::methods[] = {
-    JS_FN("plaintext", plaintext, 0, JSPROP_ENUMERATE), JS_FS_END};
+    JS_FN("plaintext", plaintext, 0, JSPROP_ENUMERATE),
+    JS_FN("rawbytes", rawbytes, 0, JSPROP_ENUMERATE), JS_FS_END};
```

Comment:
We have `fromBytes` and `rawbytes`: would it be reasonable to match the camel-case of `fromBytes`, or is there a js convention for using `rawbytes` for these sorts of methods?

Reply:
My thinking here was to follow in the `plaintext` convention which isn't camel case either. In the name of acting like this API shape wasn't a mistake and was always planned :P We can certainly switch to `rawBytes` too.

Reply:
Oh `plaintext` is a noun and not a compound word! Okay then `rawBytes` makes much more sense, thanks.